packages/twenty-front/src/modules/workspace/components/WorkspaceInviteTeam.tsx

```diff
@@ -79,16 +79,30 @@ export const WorkspaceInviteTeam = () => {
     },
   });
 
-  const submit = handleSubmit(async (data) => {
-    const emailsList = sanitizeEmailList(data.emails.split(','));
-    const result = await sendInviteLink({ variables: { emails: emailsList } });
-    if (isDefined(result.errors)) {
-      throw result.errors;
+  const submit = handleSubmit(async ({ emails }) => {
+    const emailsList = sanitizeEmailList(emails.split(','));
+    const { data } = await sendInvitation({ emails: emailsList });
+    if (isDefined(data)) {
+      if (data.sendInviteLink.result.length > 0) {
+        enqueueSnackBar(
+          `${data.sendInviteLink.result.length} invitations sent`,
+          {
+            variant: SnackBarVariant.Success,
+            duration: 2000,
+          },
+        );
+      }
+      if (!data.sendInviteLink.success) {
+        data.sendInviteLink.errors.forEach((error) => {
+          enqueueSnackBar(error, {
+            variant: SnackBarVariant.Error,
+            duration: 5000,
+          });
+        });
+      }
+    } else {
+      return;
     }
-    enqueueSnackBar('Invite link sent to email addresses', {
```

Comment:
not related to this PR but if feel duration should be a default value + allowed values should be stored in a constant, this would standardize the behavior